src/commands/api-mesh/__tests__/get.test.js

```diff
@@ -402,4 +413,127 @@ describe('get command tests', () => {
 	});
+
+	// Active flag test cases
+	test('should get last successfully mesh config with --active flag', async () => {
+		const meshId = 'dummy_meshId';
+		getMeshId.mockResolvedValueOnce(meshId);
+		getMesh.mockResolvedValueOnce({
+			meshId: 'dummy_meshId',
+			mesh: mockGetMeshConfig,
+		});
+
+		parseSpy.mockResolvedValueOnce({
+			args: {},
+			flags: {
+				ignoreCache: mockIgnoreCacheFlag,
+				active: true,
+			},
+		});
+
+		const runResult = await GetCommand.run();
+
+		expect(getMesh).toHaveBeenCalledWith(
+			selectedOrg.code,
+			selectedProject.id,
+			selectedWorkspace.id,
+			selectedWorkspace.title,
+			meshId,
+			true,
+		);
+
+		expect(runResult).toBeDefined();
+		expect(runResult.meshId).toBe('dummy_meshId');
+	});
+
+	test('should get last successfully mesh config with shorthand -a flag', async () => {
+		const meshId = 'dummy_meshId';
+		getMeshId.mockResolvedValueOnce(meshId);
+		getMesh.mockResolvedValueOnce({
+			meshId: 'dummy_meshId',
+			mesh: mockGetMeshConfig,
+		});
+
+		parseSpy.mockResolvedValueOnce({
+			args: {},
+			flags: {
+				ignoreCache: mockIgnoreCacheFlag,
+				active: true, // -a flag also sets active to true
+			},
+		});
+
+		const runResult = await GetCommand.run();
+
+		expect(getMesh).toHaveBeenCalledWith(
+			selectedOrg.code,
+			selectedProject.id,
+			selectedWorkspace.id,
+			selectedWorkspace.title,
+			meshId,
+			true,
+		);
+
+		expect(runResult).toBeDefined();
+		expect(runResult.meshId).toBe('dummy_meshId');
+	});
+
+	test('should handle NoActiveDeploymentFound error when using --active flag', async () => {
+		const meshId = 'dummy_meshId';
+		getMeshId.mockResolvedValueOnce(meshId);
+		getMesh.mockRejectedValueOnce(new Error('NoActiveDeploymentFound'));
+
+		parseSpy.mockResolvedValueOnce({
+			args: {},
+			flags: {
+				ignoreCache: mockIgnoreCacheFlag,
+				active: true,
+			},
+		});
+
+		const runResult = GetCommand.run();
+
+		return runResult.catch(err => {
+			expect(err.message).toMatchInlineSnapshot(
+				`"No active deployment found for mesh dummy_meshId. Check the mesh ID and try again or try without the --active flag. RequestId: dummy_request_id"`,
+			);
+			expect(getMesh).toHaveBeenCalledWith(
+				selectedOrg.code,
+				selectedProject.id,
+				selectedWorkspace.id,
+				selectedWorkspace.title,
+				meshId,
+				true,
+			);
+		});
+	});
+
+	test('should handle mesh not found when using --active flag', async () => {
+		const meshId = 'dummy_meshId';
+		getMeshId.mockResolvedValueOnce(meshId);
+		getMesh.mockResolvedValueOnce(null); // Mesh not found
+
+		parseSpy.mockResolvedValueOnce({
+			args: {},
+			flags: {
+				ignoreCache: mockIgnoreCacheFlag,
+				active: true,
+			},
+		});
+
+		const runResult = GetCommand.run();
+
+		return runResult.catch(err => {
+			expect(err.message).toMatchInlineSnapshot(
+				`"No active deployment found for mesh dummy_meshId. Check the mesh ID and try again or try without the --active flag. RequestId: dummy_request_id"`,
```

Comment:
Is this the error message we should expect? I thought if the mesh couldn't be found at all (e.g. it's been deleted), we'd get a more generic "mesh not found" message.

If we're mocking the return value from getMesh to `null`, then I don't believe an error will be thrown. Can you double-check this?